src/implementation/fs/index.ts

```diff
@@ -0,0 +1,27 @@
+import * as H5P from '../../';
+
+import { IContentStorage } from '../..';
+
+import DirectoryTemporaryFileStorage from './DirectoryTemporaryFileStorage';
+import FileContentStorage from './FileContentStorage';
+import FileLibraryStorage from './FileLibraryStorage';
+import InMemoryStorage from './InMemoryStorage';
+
+export default function h5pfs(
+    editorConfig: H5P.IEditorConfig,
+    librariesPath: string,
+    temporaryStoragePath: string,
+    contentPath: string,
+    contentStorage?: IContentStorage
+): H5P.H5PEditor {
+    return new H5P.H5PEditor(
+        new InMemoryStorage(),
+        editorConfig,
+        new FileLibraryStorage(librariesPath),
+        contentStorage || new FileContentStorage(contentPath),
+        new H5P.TranslationService(H5P.englishStrings),
+        (library, file) =>
+            `${librariesPath}/${library.machineName}-${library.majorVersion}.${library.minorVersion}/${file}`,
```

Comment:
The resolver here must a return a URL at which the library can be retrieved. This will return the local path. I don't think the resolver can be part of the fs implementation.

Reply:
You can implement it in the express middleware you're writing though.